.github/workflows/tests.yaml

```diff
@@ -0,0 +1,51 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - '*'
+    pull_request:
+      branches:
+        - '*'
+    schedule:
+        - cron: '59 23 * * 3'
+
+jobs:
+  unittests:
+    name: ${{ matrix.os }}, ${{ matrix.environment-file }}
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        environment-file: [ci/latest.yaml]
+
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v2
+
+      - name: setup micromamba
+        uses: mamba-org/provision-with-micromamba@main
+        with:
+          environment-file: ${{ matrix.environment-file }}
+          micromamba-version: 'latest'
+
+      - name: Install xyzservices
+        shell: bash -l {0}
+        run: pip install .
+        if: matrix.os != 'windows-latest'
+
+      - name: Install xyzservices
+        shell: powershell
```

Comment:
Is there a reason this is needed here for the `pip install` (we don't need to do that in other repo's in the org?)

Reply:
It failed without it (https://github.com/martinfleis/xyzservices/runs/2640141331?check_suite_focus=true) and we need to run `setup.py` to move providers JSON to `share/xyzservices` (after #18).

The initial failure may be related to the action we use (micromamba).